package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@artsy/reaction-force",
+  "name": "@artsy/reaction",
   "version": "0.0.0-development",
```

Comment:
@eessex I'm a little confused as to where consumers well get the version from if `package.json` isn't updated with the correct version.

Reply:
@l2succes the actual version is generated at deploy time using semantic versioning - #521

Reply:
Got it, thanks @orta